adaptation/bitrateGraphs.js

```diff
@@ -43,24 +43,24 @@ BitrateGraphs.prototype.init = function() {
   google.charts.load('current', { 'packages': ['corechart'] });
   google.charts.setOnLoadCallback(this.drawLineChart.bind(_this));
 
-  this.player.addEventHandler('onDownloadFinished', function(data) {
-    if (data.downloadType === 'media' && data.mimeType.indexOf('video') >= 0 && data.size > 1000) {
+  this.player.on('downloadfinished', function(data) {
```

Comment:
Use of strings is discouraged. You should use the enum instead: `bitmovin.player.PlayerEvent.DownloadFinished`

Reply:
I thought that too but than i found the sample `events/eventConstants` which was the only one where the Enum was used.

But if it is the preferred way i will change it everywhere (and delete the `eventConstants` sample?)?

Reply:
That makes sense. I think the `eventConstants` was created when we introduced the event constants in the player 😉